services/product/serverless.ts

```diff
@@ -29,7 +34,68 @@ const serverlessConfiguration: Serverless = {
         apiGateway: {
             minimumCompressionSize: 1024,
         },
-        environment: ENVIRONMENT,
+        environment: {
+            ...ENVIRONMENT,
+            SNS_TOPIC_SUCCESS_ARN: {Ref: 'SNSTopicSuccess'},
+            SNS_TOPIC_ERROR_ARN: {Ref: 'SNSTopicError'},
+        },
+        iamRoleStatements: [
+            {
+                Effect: 'Allow',
+                Action: 'sns:*',
+                Resource: {Ref: 'SNSTopicSuccess'},
+            },
+        ],
+    },
+    resources: {
+        Resources: {
+            SQSQueue: {
+                Type: 'AWS::SQS::Queue',
+                Properties: {
+                    QueueName: CATALOG_ITEMS_SQS,
+                },
+            },
+            SNSTopicSuccess: {
+                Type: 'AWS::SNS::Topic',
+                Properties: {
+                    TopicName: CREATE_PRODUCT_SNS_TOPIC_SUCCESS,
+                },
+            },
+            SNSSubscriptionSuccess: {
+                Type: 'AWS::SNS::Subscription',
+                Properties: {
+                    Endpoint: 'diktator007@gmail.com',
+                    Protocol: 'email',
+                    TopicArn: {Ref: 'SNSTopicSuccess'},
+                },
+            },
+            SNSTopicError: {
+                Type: 'AWS::SNS::Topic',
+                Properties: {
+                    TopicName: CREATE_PRODUCT_SNS_TOPIC_ERROR,
```

Comment:
The main idea was to implement different subscription by 'Filter policy' not by creating one more topic :)

Reply:
Yep, you are right